libraries/AP_IOMCU/AP_IOMCU.h

```diff
@@ -4,6 +4,7 @@
   For bootstrapping this will initially implement the px4io protocol,
   but will later move to an ArduPilot specific protocol
  */
+#pragma once
```

Comment:
This change should be in a separate commit. `AP_IOMCU: ...`

Reply:
Ok, i'll move it over to a AP_IOMCU PR.

Reply:
Same PR is fine, just needs to be in its own commit. We like to keep the commit history tidy with one per directory. 

https://ardupilot.org/dev/docs/git-interactive-rebase.html

Reply:
Understood. I have made the change, first time using rebasing pretty cool stuff actually! :)